### PR TITLE
Excel Tweaks and balance.

### PR DESCRIPTION
**Pull request by Rexdyer1**
## About The Pull Request

This will change up a bunch of armor values and add a new mothball version of the excel voidsuit to the stash to replace the normal one; this one lacks a excelsior hud and has much less armor. eventually, I hope to add new sprites for it but for now this will do. Excel armor is now buffed a bit more to encourage people to use it and showcase themselves as Excel agents. This will also add the boltgun as an easily bought weapon from the Excelsior teleporter so it finally gets used and so excels have a cheap weapon to easily defend themselves early on. It will also rebalance the cost slightly for the Excelsior stock parts and circuits to make growth much faster so it doesn't take so long for excels to expand their base. excelsior jumpsuits are now much more heavily armor to encourage their use by excels. 

## Why It's Good For The Game

The whole point of this update is to make it more viable for excel antags to dress and utilize Excelsior resources earlier so that they can do battle with the crew in a much timelier fashion. It makes their gear more useful and their jumpsuit is now armored. they have access to the bolt rifle and strips of ammo as a cheap form of weaponry from the teleporter to give them an easier means of defense. 

## Testing

Stat changes and addition of one disk. 

## Changelog
:cl:Acehunterkill
add: New excel medical disk
Balance: new stats for voidsuit
Balance: armor added to excel jumpsuits
add:New mothball variant of excel voidsuit,
tweak: new mothball excel voidsuit added to excel stash. 
tweak:price changes to excel teleporter
Balance:Boltgun and ammo added to excelsior teleporter list. 
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/design_disks/excelsior.dm [hunk 1]
```diff
@@ -196,3 +196,15 @@
 		/datum/design/autolathe/ammo/shotgun_beanbag,
 		/datum/design/autolathe/ammo/shotgun_pellet
 	)
+
+/obj/item/computer_hardware/hard_drive/portable/design/excelsior/freeheal
```
Comment (Humonitarian) [inline code/game/objects/items/weapons/design_disks/excelsior.dm:200]: And how does Excelsior get access to biomatter? Can they order it? Does their autolathe even support it?

Comment (ValoTheValo) [reply code/game/objects/items/weapons/design_disks/excelsior.dm:200]: Their autolathe does, and they can recycle clothing into it. or just steal it from neotheo

File: code/modules/clothing/under/excelsior.dm [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
 	name = "random excelsior jumpsuit"
-	desc = "Excelsior jumpsuit designed to boost morale and spread the revolution"
+	desc = "Reject decadent capitalist 'fashion', embrace the liberation of true revolutionary style"
 	icon_state = "excelsior_white"
 	item_state = "bl_suit"
 	has_sensor = 0
 	spawn_blacklisted = TRUE
+	armor = list(
+		melee = 3,
+		bullet = 4,
```
Comment (ValoTheValo) [inline code/modules/clothing/under/excelsior.dm:10]: giving it bullet armor this high would make it the single best uniform in the game. the only other armored uniforms, Rockstar Outfit and OneStar Jumpsuit have 1 and 2 bullet armor respectively

File: code/modules/clothing/spacesuits/void/excelsior.dm [hunk 3]
```diff
@@ -92,3 +92,57 @@
...
+	)
+	siemens_coefficient = 0 //Shockproof!
+	breach_threshold = 6
+	resilience = 0.08
+	matter = list(
+		MATERIAL_PLASTIC = 30,
+		MATERIAL_STEEL = 10,
+		MATERIAL_PLASTEEL = 5
+	)
+	helmet = /obj/item/clothing/head/space/void/excelsior_ruined
```
Comment (ValoTheValo) [inline code/modules/clothing/spacesuits/void/excelsior.dm:119]: wrong path, should be /obj/item/clothing/head/space/void/excelsior_mothball

File: config/example/admin_ranks.txt [hunk 4]
```diff
@@ -1,38 +0,0 @@
-########################################################################################
```
Comment (ValoTheValo) [inline config/example/admin_ranks.txt:1]: why did you delete the admin ranks file

Comment (Rexdyer1) [reply config/example/admin_ranks.txt:1]: yeah that one was something I've been trying to figure out. so it seems to me when I did the usual testing for admin permissions it added itself onto the PR?
I think I'mma just redo the PR itself.

File: code/modules/clothing/under/excelsior.dm (hunk 2 above)
Comment (Rexdyer1) [reply code/modules/clothing/under/excelsior.dm:10]: yeah I was considering the effects of that...it would be best to drop it to 2 maybe, I just thought it'd be a good way to incentivise people using the excel jumpsuits.

File: code/game/objects/items/weapons/design_disks/excelsior.dm (hunk 1 above)
Comment (Rexdyer1) [reply code/game/objects/items/weapons/design_disks/excelsior.dm:200]: yeah, I wanted to give excels something to utilize their biomatter capabilities early on. I am considering replacing the bandages and ointments with Excel specific healing packs but I need to figure out the spriting for that.

File: config/example/admin_ranks.txt (hunk 4 above)
Comment (Rexdyer1) [reply config/example/admin_ranks.txt:1]: Well unless there is a way to remove that. because the admin files were completely unintentional and a fluke. I just can't figure out how to undo it from here without redoing the PR.

Comment (Rexdyer1) [reply config/example/admin_ranks.txt:1]: but yeah that is completely a bug.
